src/app/pages/add-contact/addContact.component.ts

```diff
@@ -31,30 +31,49 @@ export class AddContactComponent implements OnInit {
     }
 
   isEmailInValid: boolean = true;
-  invalidAddContact: boolean = false;
+  isMobileNumberInValid: boolean = true;
```

Comment:
I just noticed this, but you spelt invalid wrong all these three variables. The v should not be capitalized.